src/constant.ts

```diff
@@ -0,0 +1,2 @@
+export const TFS_URL = "http://tfs.sys.dom:8080/tfs/DefaultCollection";
+export const SaxoTrader_Project_TFS_ID = "de3a9280-eeb1-4d1f-b391-956cb5eee690";

```

Comment:
would be great to rename this to SaxoTrader_Project_TFS_REPO_ID

Reply:
done